docs/book/v3/application-integration/usage-in-a-laminas-mvc-application.md

```diff
@@ -37,7 +37,22 @@ return [
 ];
 ```
 
-The factory `Laminas\Cache\Service\StorageCacheAbstractServiceFactory` uses the configuration, searches for the configuration key `caches` and creates the storage adapters using the discovered configuration.
+## Register a storage adapter
```

Comment:
This is the wrong place for this description as it is unrelated to the rest of the content and is incomplete.
See also: https://github.com/laminas/documentation/issues/8

Please create a new page for this topic and add a complete example that can be fully reproduced by the reader.